python/hsfs/feature_store.py

```diff
@@ -986,30 +992,25 @@ def get_or_create_spine_group(
                                 )
             ```
 
-        Note that you can inspect the dataframe in the spine group, or replace the dataframe:
+        Note that you can inspect the DataFrame in the spine group, or replace the DataFrame:
 
         ```python
         spine_group.dataframe.show()
 
         spine_group.dataframe = new_df
         ```
 
-        The spine can then be used to construct queries, with only one speciality:
-
         !!! note
             Spines can only be used on the left side of a feature join, as this is the base
             set of entities for which features are to be fetched and the left side of the join
             determines the event timestamps to compare against.
 
-        **If you want to use the query for a feature view to be used for online serving,
-        you can only select the label or target feature from the spine.**
-        For the online lookup, the label is not required, therefore it is important to only
-        select label from the left feature group, so that we don't need to provide a spine
-        for online serving.
+        !!! note
+            Spine Groups are not currently supported for online serving.
```

Comment:
I think this is misleading. Spine feature groups have no meaning in serving as they represent labels and the online feature store only store the most recent version of the data.